core/tools.js

```diff
@@ -979,6 +979,10 @@
 					calculator = CKEDITOR.dom.element.createFromHtml( '<div style="position:absolute;left:-9999px;' +
 						'top:-9999px;margin:0px;padding:0px;border:0px;"' +
 						'></div>', CKEDITOR.document );
+				}
+
+				// (#5158)
+				if ( calculator.isDetached() ) {
```

Comment:
I think that at this point we can assume that calculator has been somehow manipulated without our supervision. So, we can assume that it's broken and should be re-created instead.